Add 21.04 support in .travis.yml

kjuanman opened the pull request:
add ubuntu hirsute (21.04) support. Fix #818

gfrenoy commented:
Thanks for your contribution !!

It looks like "hirsute" is now available in the "docker hub" of packpack (see https://hub.docker.com/r/packpack/packpack/tags/?page=1&ordering=last_updated) so you should remove `DOCKER_REPO=asbru/asbru-cm`, it is not needed :)

We can actually also remove it for groovy as well.

kjuanman commented:
> Thanks for your contribution !!
> 
> It looks like "hirsute" is now available in the "docker hub" of packpack (see https://hub.docker.com/r/packpack/packpack/tags/?page=1&ordering=last_updated) so you should remove `DOCKER_REPO=asbru/asbru-cm`, it is not needed :)
> 
> We can actually also remove it for groovy as well.

Thanks for the explanation! 
I removed DOCKER_REPO in ubuntu hirsute and groovy. And Added debian bullseye, which is available in packpack too.

gfrenoy commented:
Looks good, thanks :)  Let's give it a try ...
